### DISCUSSION TITLE
Support for later versions of caaa?

### DISCUSSION THREAD

**Discussion by gverran** [prowide/prowide-iso20022#65] [Q&A] [answered]
Hi @zubri,

I logged an issue for this, but can you please upgrade caaa to include subsequent versions from the version that is packaged?  I really hope that you'll be able to upgrade any other ISO-20022 messages to include versions not present?  I see there is a comment regarding pain.001 below.

Looking forward to your response.

~Glen

**Comment by prshankar** [comment]
Hello Team,
I am looking at MxCaaa01300108 and MxCaaa01300109 and some more of recent versions for my development. Any idea on what is the expected time frame.

**Reply by zubri** [reply]
Added in release SRU2022-9.3.4 along many other missing types

**Comment by zubri** [comment]
We are working on it. Should be done by next week.

**Comment by zubri** [comment, marked as answer]
Added in release SRU2022-9.3.4 along many other missing types

**Reply by gverran** [reply]
@zubri Thanks very much!! :)  How long does it take to propagate to Mvn Repository, etc?

**Reply by zubri** [reply]
2 days at most
